src/pages/Home.jsx

```diff
@@ -1,22 +1,50 @@
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from "@ionic/react";
-import ExploreContainer from "../components/ExploreContainer";
-import "./Home.css";
+import { useState } from "react";
+import { IonContent, IonPage, IonGrid, IonRow, IonCol } from "@ionic/react";
+
+import "./Home.scss";
+import DeleteButton from "../components/button/DeleteButton";
+import FollowButton from "../components/button/FollowButton";
+import UnfollowButton from "../components/button/UnfollowButton";
+import Toast from "../components/toast/Toast";
+import HomeToolbar from "../components/toolbar/HomeToolbar";
+import TextInput from "../components/text-input/TextInput";
+import { googleLogin } from "../services/google-auth";
 
 const Home = () => {
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+  const [name, setName] = useState("");
+
+  const handleLogin = () => {
+    googleLogin();
+  };
```

Comment:
No need for `handleLogin` here right? Unless this is just to test the login function?

Reply:
The code on Home.jsx is just to show what components there are. This is a use case for the button:

```
const handleGoogleLogin = async (googleAuthStatus) => {
    if (googleAuthStatus === GoogleAuthStatus.GOOGLE_AUTH_SUCCESS) {
      // success
      setToastMessage("Google authentication successful!");
      setToastColor("success");
      setShowToast(true);
      history.replace("/home");
    } else {
      // error
      setToastMessage("Google authentication failed. Please try again.");
      setToastColor("danger");
      setShowToast(true);
    }
  };

  const handleGuestLogin = async () => {
    history.replace("/home");
  };
```